pennylane/transforms/tape_expand.py

```diff
@@ -28,11 +28,6 @@
     not_tape,
 )
 
-# Needed for custom decomposition context manager
-from pennylane.transforms.qfunc_transforms import NonQueuingTape
-
-NonQueuingTape = type("NonQueuingTape", (NonQueuingTape, qml.tape.QuantumTape), {})
-
```

Comment:
So these were just not used by the file? 😒

Reply:
hehe. they were used once upon a time, but the usage was removed (and the definition here was not).